spec/unit/libraries/splunk_password_spec.rb

```diff
@@ -6,12 +6,28 @@
 
 describe 'CernerSplunk::splunk_password' do
   let(:splunk_secret) { 'qYFEHts8G0E/ABbp' }
-  let(:password) { 'password' }
+  let(:password) { 'password' || '$1$password' || '$7$password' }
 
   describe '.splunk_encrypt_password' do
     context 'when the password has already been encrypted' do
       let(:password) { '$1$RhLQiUyG3Qc7' }
 
+      it 'does not re-encrypt it again' do
+        encrypted = CernerSplunk.splunk_encrypt_password(password, splunk_secret)
+        expect(encrypted).to eq(password)
+      end
+    end
+    context 'when the password has already been encrypted' do
```

Comment:
This is basically a duplicate of the test above on line 12.  It can be deleted.

Reply:
Removed, thanks for pointing that out: https://github.com/cerner/cerner_splunk/pull/237/commits/6a27ef4c7dc52bbfb889dfceaf867d131f7262ba